actionview/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   `translate` now allows `default: [[]]` again for a default value of `[]`.
+
+    Fixes a regression introduced in 4.2.1.
+
+    See #19640 and the fix in #19649.
```

Comment:
Oh interesting, thanks for the info. I included this because I tend to find it really difficult to track down issues and pull requests for stuff mentioned in the changelogs.